sklearn/datasets/base.py

```diff
@@ -255,14 +255,22 @@ def load_iris():
 
     Read more in the :ref:`User Guide <datasets>`.
 
+    Parameters
+     ----------
+    return_X_y : boolean, default=False.
+        If True, returns (data, target) instead of a Bunch object.
+        See below for more information about the `data` and `target` object.
+
     Returns
     -------
     data : Bunch
         Dictionary-like object, the interesting attributes are:
         'data', the data to learn, 'target', the classification labels,
         'target_names', the meaning of the labels, 'feature_names', the
         meaning of the features, and 'DESCR', the
-        full description of the dataset.
+        full description of the dataset. This is test text.
```

Comment:
Hey @amueller, thanks for your help.

Yes, my dev version is imported when I try doing that. Also, I ran ```pip list``` and only the dev version installation is shown in the list. 

![image](https://cloud.githubusercontent.com/assets/12952080/17220420/c3a6c2c0-550c-11e6-9e41-4850953186a6.png)